guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>You can use the <code>prepare_tabulardata()</code> method to create a <code>TabularDataObject</code> that formats the sequential data so that it can be fed into the <code>TimeSeriesNetwork</code> model. It allows you to use attribute data from feature layers and <code>DataFrames</code>. Additionally, you can use explanatory rasters. The raster can be an imagery layer that can be accessed through ArcGIS Enterprise or through local raster files accessed using the <code>Raster</code> class from the <code>arcgis.raster</code> module. This method performs normalization of the datasets, imputation, and splits it into training and validation datasets, before then feeding the datasets into the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>